sdk/src/main/scala/com/horizen/storage/AbstractHistoryStorage.scala

```diff
@@ -271,9 +274,7 @@ class AbstractHistoryStorage[PM <: SidechainBlockBase[_ <: Transaction, _ <: Sid
     this
   }
 
-  def getFeePaymentsInfo(blockId: ModifierId): Option[FeePaymentsInfo] = {
-    storage.get(feePaymentsInfoKey(blockId)).asScala.flatMap(baw => FeePaymentsInfoSerializer.parseBytesTry(baw.data).toOption)
-  }
+  def getFeePaymentsInfo(blockId: ModifierId): Option[FPI]
```

Comment:
Probably we may pass `PaymentsInfoSerializer` to the constructor, similar to how we passed `blockSerializer: ScorexSerializer[PM],`
then will have a generic implementation as it was before.

Reply:
Ok, done